CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ Currently the versioning policy of this project follows [Semantic Versioning v2.
 
 ## Unreleased - 2018-??-??
 
+### Fixed
+
+* Remove unused field in StandardTypeManager. ([#667](https://github.com/spotbugs/spotbugs/pull/667))
```

Comment:
@KengoTODA : WDYT? IMHO changelog is for important or end-user visible changes, not for house keeping.

Reply:
I also think this change is not important.  I removed.